airflow/providers/elasticsearch/CHANGELOG.rst

```diff
@@ -27,6 +27,17 @@
 Changelog
 ---------
 
+5.1.0
```

Comment:
```suggestion
6.0.0
```

Since we are suggesting a breaking change, we need to have a major bump

Reply:
Ok - will revise accordingly